chainconsumer/plotter.py

```diff
@@ -4,12 +4,13 @@
 from matplotlib.ticker import MaxNLocator, ScalarFormatter
 from matplotlib.textpath import TextPath
 from numpy import meshgrid
-from scipy.interpolate import interp1d
+from scipy.interpolate import interp1d, interp2d
 from scipy.ndimage import gaussian_filter
 from scipy.stats import norm
 
 from .helpers import get_extents, get_smoothed_bins, get_grid_bins
 from .kde import MegKDE
```

Comment:
This should now be unused, ya?

Reply:
Actually I'm using fastKDE with default settings, so it outputs the pdf and the "optimal" points it used to estimate it (referred to as "axes" in the code). Then I'm interpolating between those points, both in 1D and 2D. Setting the `axes` keyword of `fastKDE.pdf` returns strange results. But it might be worth understanding how to do that without interpolating. I'll give it a try.

Reply:
Nah we'd always need to interpolate, because the grid that comes out will be a power of 2 and have tons of spacing (for efficient FFT and smooth FFT results respectively), we just need to make sure that the grid density in the region of interest (out to 3 sigma or so) is well sampled by the supplied axes.